### DISCUSSION TITLE
HRR at a ventilation duct

### DISCUSSION THREAD

**Discussion by LucilePl** [firemodels/cfast#2126] [Q&A]
Does CFAST predict combustion at a ventilation duct as it does at the vents (HRR Vent Jet Fire) ?

**Comment by preneke** [comment]
No, CFAST doesn't predict fires at ventilation vents.

**Comment by rpeacoc** [comment]
CFAST calculates door-jet fires like it does “regular” fires except that the entrainment for the door-jet fire plume is calculated differently. Details should be in the Technical Reference Guide.

**Reply by rpeacoc** [reply]
My mistake. Vent fires are not done at HVAC vents. The unburned fuel is transported and may burn in the upper layer (if there is a fire in that room) or as a door jet fire at a horizontal flow vent in the room.
